src/Checkbox.jsx

```diff
@@ -96,23 +93,17 @@ export default class Checkbox extends React.Component {
     const state = this.state;
     const prefixCls = props.prefixCls;
     let checked = state.checked;
-    if (typeof checked === 'boolean') {
-      checked = checked ? 1 : 0;
-    }
-    const className = classNames({
-      [props.className]: !!props.className,
-      [prefixCls]: 1,
+
+    const className = classNames([prefixCls], {
+      [props.className]: props.className,
       [`${prefixCls}-checked`]: checked,
-      [`${prefixCls}-checked-${checked}`]: !!checked,
       [`${prefixCls}-focused`]: state.focus,
       [`${prefixCls}-disabled`]: props.disabled,
     });
     return (
-      <span
-        className={className}
-        style={props.style}
-      >
+      <span className={className} style={props.style}>
           <span className={`${prefixCls}-inner`} />
+
```

Comment:
Just personal style,I think it's more clear to read,and I see some libs or docs do the same.So,if your team don't like it,I will remove this later